operations/nhc.rst

```diff
@@ -17,6 +17,26 @@ match your setup.
 When NHC identifies a node to be unhealthy, NHC drains this node to prevent
 future jobs from running on it.
 
+Acquire and Provide NHC
+=======================
+
+Before the ``slurmd`` charm can run NHC must be installed, the ``.tar.gz`` must be
+supplied:
+
+Acquire NHC
+------------
+
+.. code-block:: bash
+    
+     $ wget https://github.com/mej/nhc/releases/download/1.4.3/lbnl-nhc-1.4.3.tar.gz
+
+Providing NHC
+-------------
+
+.. code-block:: bash
+     
+     $ juju deploy slurmd --resource nhc=lbnl-nhc-1.4.3.tar.gz
+
```

Comment:
This section should explain _when_ it is needed to do that. If getting the charms from Charmhub, this step is not needed, unless the person wants to change the version used. Thoughts?

Reply:
Please see [c20fbfa](https://github.com/omnivector-solutions/osd-documentation/pull/77/commits/c20fbfaef948698a0d8088fdeaf5dbdc2bd103d2) and let me know if that looks good.